capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -14,6 +14,13 @@ class ClubPage extends Component {
         .then(response => response.json()).then(res => this.setState({club: res[0]}));
   }
 
+  handleClick = (e) => {
+    e.preventDefault();
+    fetch(`/api/clubs?id=${this.props.match.params.id}`, {method: "delete"})
+    .then(response => {console.log(response)});
+    this.props.history.push("/myclubs");
```

Comment:
This should probably happen only once you know the delete was successful, though that change should also be paired with some sort of user-facing error message when it was unsuccessful.

Reply:
Will move the history push into the `then` clause and add a `catch` as discussed above. For now we'll simply use a javascript alert to signify a failed delete. In the future the exponential backoff retry would be a great solution to the problem.